doc/getting-started/how-helix-works.md

```diff
@@ -0,0 +1,19 @@
+# How does Helix work?
+
+This page takes a behind-the-scenes look at the overall Helix architecture and describes its main components.
+
+![Helix Architecture Overview](/assets/helix-overview.png)
+
+Helix uses the Fastly CDN to deliver content and deploys Helix-specific _VCL edge code_ to process client requests.
+
+That VCL code is generated by the _hlx publish_ command along with the _Helix Publish Service_ deployed in Adobe I/O Runtime.
+Together, they generate and setup the appropriate Fastly configurations based on the contents
+of the source GitHub repository which provides the content, rendering templates and code as well as the 
+Helix configuration file.
+
+The _hlx deploy_ command is used to deploy the rendering code in Adobe I/O Runtime, based on the rendering templates and code
+supplied in the GitHub source repository.
+
+Updates to these components are atomic, and the Helix Strains let users define staged variants of their content, to have
+full control on the publishing process and allow for intermediate reviews before publishing content.
```

Comment:
I'm not sure we should advertise this 'publishing process' at this point because I don't think we've converged on a recommended practice for it yet. If we have and I just don't know it, great, but then we should document it 😄

Reply:
Makes sense, removing this!